README.md

```diff
@@ -94,7 +94,7 @@ services:
       # https://[wakapi-instance]/api/compat/wakatime/v1/users/current/heartbeats.bulk
       HAKA_REMOTE_WRITE_TOKEN: "<token>"
     ports:
-      - "127.0.0.1:8080:8080"
+      - "8080:8080"
```

Comment:
Could you add a comment saying this that this will start the services on all interfaces just to be explicit?

Reply:
Should I include a link to the documentation as well? 

I plan on including this: https://github.com/compose-spec/compose-spec/blob/master/spec.md#ports